src/map/unit.cpp

```diff
@@ -1840,30 +1847,109 @@ void unit_set_attackdelay(block_list& bl, t_tick tick, e_delay_event event)
 	if (ud == nullptr)
 		return;
 
+	uint16 attack_delay = 0;
+	uint16 act_delay = 0;
```

Comment:
```suggestion
	t_tick attack_delay = 0;
	t_tick act_delay = 0;
```

Reply:
The return type of status_get_adelay and status_get_amotion is uint16, that's why I'm using it here. Why reserve more memory when not needed?